src/faction_camp.cpp

```diff
@@ -1675,6 +1677,8 @@ void basecamp::worker_assignment_ui()
         } else {
             mvwprintz( w_followers, point( 1, 4 ), c_light_red, no_npcs );
         }
+        mvwprintz( w_followers, point( 1, FULL_SCREEN_HEIGHT - 2 ), c_light_gray,
+                   _("Press i to inspect this follower."));
```

Comment:
Use `ctxt.get_desc( "<ACTION>" )` to get the keybinding description on the fly.

Reply:
Done, thx